feedbooks.py

```diff
@@ -169,6 +181,41 @@ def improve_description(self, id, metadata):
 
         return metadata
 
+    def replace_css(self, representation):
+        """This function will replace the content of every CSS file listed in an epub's
+        manifest with the value in self.new_css. The rest of the file is not changed.
+        """
+
+        if representation.media_type == Representation.EPUB_MEDIA_TYPE:
+
+            old_zip_content = StringIO(representation.content)
+            new_zip_content = StringIO()
+
+            with ZipFile(old_zip_content) as old_zip:
+                with old_zip.open("META-INF/container.xml") as container_file:
+                    container = container_file.read()
+                    rootfile_element = etree.fromstring(container).find("{urn:oasis:names:tc:opendocument:xmlns:container}rootfiles").find("{urn:oasis:names:tc:opendocument:xmlns:container}rootfile")
```

Comment:
This will crash if the first find() call finds nothing. If we can't find what we're looking for at any point, we should leave the content alone, possibly logging a warning.